src/main/java/Duke.java

```diff
@@ -1,10 +1,294 @@
+import task.*;
+import storage.Storage;
+import ui.Ui;
+
+import java.util.*;
+
 public class Duke {
+
+    private Storage storage;
+    private TaskList taskList;
+    private Ui ui;
+
+    public Duke() {
+
+        this.ui = new Ui();
+        this.storage = new Storage(System.getProperty("user.dir") + "/data/", "duke.txt");
+        this.taskList = storage.load();
+
+    }
+
+    public void run() {
+
+        Scanner sc = new Scanner(System.in);
```

Comment:
As above, perhaps some of the extra lines between statements can be removed and  the statements can be grouped instead. What do you think?

Reply:
Changed :)